modules/op-authenticating-to-an-oci-registry.adoc

```diff
@@ -0,0 +1,49 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-chains-for-pipelines-supply-chain-security.adoc
+
+:_content-type: PROCEDURE
+[id="authenticating-to-an-oci-registry_{context}"]
+= Authenticating to an OCI registry
+
+Before pushing signatures to an OCI registry, cluster administrators must configure Tekton Chains to authenticate with the registry. The Chains controller uses the same service account under which the task runs execute. To set up a service account with the necessary credentials for pushing signatures to an OCI registry, perform the following steps:
```

Comment:
Last sentence: IBM SG, avoid self referential language, like "following these steps/this procedure" 

`What do you think about a rephrase like this: `Set up a service account with the necessary credentials for pushign signatures to an OCI registry.` 

.Procedure